app/helpers/application_helper.rb

```diff
@@ -1,5 +1,25 @@
 module ApplicationHelper
-  def ficon(name, size = nil)
-    content_tag(:i, nil, class: "fi-#{name} #{size} icon")
+  def ficon(name, size = nil, *classes)
+    add = classes.map(&:to_s).join(' ')
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.